public/views/settings.html

```diff
@@ -0,0 +1 @@
+Some settings go here
```

Comment:
yup - if youre an admin, you'll have admin settings showing up. if youre staff, it'll just be to change some stuff on your account (at the very least, changing your password)